### PR TITLE
Accessability improvements

### PR DESCRIPTION
**Pull request by mfix-stripe**
image

### CODE REVIEW

File: pages/index.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ description: From personal blogs to massive documentation sites, Markdoc is a co
 
   {% item %}
 
-  #### Developer & writer friendly {% .jumbo %}
+  ### Developer & writer friendly {% .h4 .jumbo %}
```
Comment (matv-stripe) [inline pages/index.md:41]: feels odd to me we are giving h3s a h4 class. Why do we need to do this?

File: public/globals.css [hunk 2]
```diff
@@ -269,7 +269,7 @@ h3.jumbo {
   line-height: 52px;
 }
 
-h4.jumbo {
+h4.jumbo, .h4.jumbo {
```
Comment (matv-stripe) [inline public/globals.css:272]: Did you mean to add this?

**Review by matv-stripe — CHANGES_REQUESTED** [review]
quick question @mfix-stripe

File: pages/index.md (hunk 1 above)
Comment (mfix-stripe) [reply pages/index.md:41]: This is a super common CSS pattern. It basically means that semantically, we want this element to be an h3, but _styled_ as an h4.

File: public/globals.css (hunk 2 above)
Comment (mfix-stripe) [reply public/globals.css:272]: Yup 🙂

File: pages/index.md (hunk 1 above)
Comment (mfix-stripe) [reply pages/index.md:41]: I can look at alternatives though 👍

**Comment by mfix-stripe** [comment]
Fixed the .h4 thing, thanks @matv-stripe

File: pages/index.md (hunk 1 above)
Comment (matv-stripe) [reply pages/index.md]: Ah, got it

**Pull request merged**